### ISSUE TITLE
Doc updates for V1 getting started

### ISSUE THREAD

**Issue by EdMoffatt** [hyperledger/composer#667]
Updates to the doc walking users through the newly updated V1 Getting Started.

**Comment by mahoney1** [comment]
Updated the docs (Quickstart.md) and executed a pull request for all changes...waiting to merge once Travis completes.

**Comment by davidkel** [comment]
@mahoney1 I think you need to check with @sstone1 and @dselman as to how getting started is to be provided. The change to getting started to hlfv1 was reverted as it was causing confusion and problems for users.
There is a separate repository created for getting started specifically for hlf v1 called 'sample-applications-hlfv1' currently as I assume that there was a plan to keep the 0.6 version going while making available a hlf v1 version for the community to have a go with.
I've made changes to this project to make it easier to use. There is no need now to mess around manually creating connection profiles, it does it all for you. There is a new script called 'createProfile.sh' that gets run it does the following
- clears any existing profile and keystore
- creates a connection profile called 'hlfv1'
So the way a user works with the getting started sample for hlf v1 is identical to the way they do it in the v0.6 sample. They just run npm install (this also does the npm network list so don't know if you want to still include that. If you do the command is
```
composer network list -n digitalproperty-network -p hlfv1 -i admin -s adminpw
```
In summary the differences between the original getting-started project and the new getting-started project in the git repository sample-applications-hlfv1 are
- The script actions which you've already updated (apart from the createProfile.sh which is the new one I added). 
- the profile name is called hlfv1
- the credentials directory is .hfc-key-store in the users home directory
- the enrollSecret is adminpw
